src/main/java/io/javalin/metrics/JavalinMetrics.java

```diff
@@ -0,0 +1,32 @@
+package io.javalin.metrics;
+
+import io.micrometer.core.instrument.Metrics;
+import io.micrometer.core.instrument.binder.jetty.JettyServerThreadPoolMetrics;
+import io.micrometer.core.instrument.binder.jetty.JettyStatisticsMetrics;
+import io.micrometer.core.instrument.composite.CompositeMeterRegistry;
+import org.eclipse.jetty.server.handler.StatisticsHandler;
+import org.eclipse.jetty.util.thread.ThreadPool;
+import org.jetbrains.annotations.NotNull;
+
+import java.util.Collections;
+
+public class JavalinMetrics {
```

Comment:
Could this be Kotlin?

Reply:
I thought about that one too. I could change it to Kotlin if you want.

What about the possible java rewrite plans?

Reply:
Even if we might rewrite to Java we should still follow the current convention, which is to use Kotlin everywhere except for certain public APIs where Java is required.